examples/src/main/java/alluxio/cli/MiniBenchmark.java

```diff
@@ -64,18 +64,17 @@ private static void usage() {
    * @throws Exception if error occurs during tests
    */
   public static void main(String[] args) throws Exception {
-    if (args.length != 4) {
+    if (args.length != 3) {
       usage();
       System.exit(-1);
     }
-    AlluxioURI masterLocation = new AlluxioURI(args[0]);
-    Configuration.set(PropertyKey.MASTER_HOSTNAME, masterLocation.getHost());
-    Configuration.set(PropertyKey.MASTER_RPC_PORT, masterLocation.getPort());
+    Configuration.set(PropertyKey.MASTER_HOSTNAME, Configuration.get(PropertyKey.MASTER_HOSTNAME));
```

Comment:
71 and 72 are no-op---you get a value and set it back. It is fine to remove these two lines